finagle/h2/src/main/scala/io/netty/handler/codec/http2/H2FrameCodec.scala

```diff
@@ -210,7 +210,6 @@ object H2FrameCodec {
       val fr = new Http2InboundFrameLogger(new DefaultHttp2FrameReader, frameLogger)
       new DefaultHttp2ConnectionDecoder(conn, encoder, fr)
     }
-    decoder.localSettings(settings)
```

Comment:
@olix0r: this API has been removed.  what was the intention behind setting the decoder's local settings?  does removing this cause any problems?